cli/src/main/scala/org/apache/celeborn/cli/config/CliConfigManager.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.cli.config
+
+import java.io.{File, PrintWriter}
+
+import scala.io.Source
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+
+import org.apache.celeborn.cli.config.CliConfigManager.cliConfigFilePath
+import org.apache.celeborn.common.util.Utils
+
+case class CliConfig(@JsonProperty("cliConfigData") cliConfigData: Map[String, String])
+
+object CliConfigManager {
+  val cliConfigFilePath =
+    s"${sys.env.getOrElse("CELEBORN_CONF_DIR", sys.env("HOME"))}/celeborn-cli.conf"
+}
+
+class CliConfigManager {
+
+  private val mapper = new ObjectMapper()
```

Comment:
Is there any particular reason for changing the configuration format to JSON? JSON format is not as user-friendly as the properties file.

Reply:
sure, let me make it a regular conf file. This is not really directly accessed by the user, it should only be accessed via the cli command tools.